tests/python/tank_test/tank_test_base.py

```diff
@@ -809,12 +902,25 @@ def _move_data(path):
             if os.path.exists(db_path):
                 print('Removing db %s' % db_path)
                 # Importing pdb allows the deletion of the sqlite db sometimes...
-                import pdb
+                import pdb # noqa
```

Comment:
_# Importing pdb allows the deletion of the sqlite db sometimes..._, hmmm... that seems very mysterious and magical... any valid reason which would explain it?

Reply:
No idea. I'm not the one who added this. Maybe it's worth removing it and see what happens? :D